### DISCUSSION TITLE
2 window managers same time?

### DISCUSSION THREAD

**Discussion by hamburger-fries** [i3/i3#4748] [Q&A]
Is there an install script for setting up i3 + KDE Plasma v5? 

Thanks

**Comment by Airblader** [comment]
You cannot have two window managers, and KDE isn't a window manager; it's a desktop environment.

I don't know of a script per se, but there are some resources like this:

https://github.com/heckelson/i3-and-kde-plasma

**Comment by ghost** [comment]
Why would you want to use two WM's O_o This defeats the purpose of i3wm. If you configure your i3wm correctly you don't need any other window manger. I'm fairly new to i3wm only using it since April, 2021, I used XFCE for the past 10-years and wish I switched to i3wm sooner :-) LOVE i3WM, NEVER GOING BACK!
